packages/metadata-common/src/MetadataWidget.tsx

```diff
@@ -115,7 +115,7 @@ export class MetadataDisplay<
     return showDialog({
       title: `Delete ${
         this.props.labelName ? this.props.labelName(metadata) : ''
-      } '${metadata.display_name}'?`,
+      } ${this.props.schemaType || ''} '${metadata.display_name}'?`,
```

Comment:
Not really following this syntax either.  This seems to place `schemaType` BEFORE the instance's display name.

Could you provide some examples of what follows "Delete " here?

Reply:
`labelName` is only provided by the runtimes widget, and returns the schema title.
An example of deleting a runtime config instance text:
`Delete Kubeflow Pipelines runtime configuration 'Trills KFP Runtime'?`
For a component catalog:
`Delete component catalog 'KFP Preloaded Components - URL'?`